python/dgl/distributed/partition.py

```diff
@@ -1542,7 +1597,7 @@ def gb_convert_single_dgl_partition(
     )
     orig_graph_path = os.path.join(
         os.path.dirname(part_config),
-        part_meta[f"part-{part_id}"]["part_graph"],
+        part_meta[f"part-{part_id}"]["node_feats"],
```

Comment:
this is not the original graph path though it works for now as we only use the dirname

Reply:
I rename the variable